CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
The `salary` field is a redundant and unused string field that appears to be a temporary placeholder. Leaving TODOs and unused fields in production code can lead to confusion and maintenance issues.

```suggestion
// TODO: Remove before merging
```